soh/soh/Enhancements/bootcommands.c

```diff
@@ -26,6 +26,8 @@ void BootCommands_Init()
     CVarClear("gOnFileSelectNameEntry"); // Clear when soh is killed on the file name entry page
     CVarClear("gBetterDebugWarpScreenMQMode");
     CVarClear("gBetterDebugWarpScreenMQModeScene");
+    CVarClear("gCheatEasyPauseBufferLastInputs");
+    CVarClear("gCheatEasyPauseBufferTimer");
```

Comment:
do these need to be cvars? i don't have a better idea for where to put them off the top of my head but cvars don't quite feel right

Reply:
This is something where a CVar rework/support for in-memory only CVars would be convenient for.

Alternatively, I could see these maybe moved to the GameInteractor state properties with C get/set methods, just would be a bit boiler plate-y to setup.

Reply:
I guess even with in-memory only cvars I'd think of them more as "settings" than "state variables," so maybe GI state would be a cool pattern to set up? I'd love to hear what other people think about this.